workflow-templates/library-checks-and-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Library checks & release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+      - alpha
+      - beta
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+jobs:
+  main:
+    name: Run checks and deploy
+    runs-on: [self-hosted, fear]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Set GitHub packages registry
+        run: |
+          npm config set '//npm.pkg.github.com/:_authToken' ${{ secrets.GH_TOKEN }}
```

Comment:
Just to double check, the `${{ secrets.GH_TOKEN }}` won't be displayed in any Github Action logs, right?

Reply:
Checking on the Karr repo, that has an action 90% similar, where we can also see this line: https://github.com/Typeform/karr/actions/runs/636123425/workflow#L33

[There](https://github.com/Typeform/karr/runs/2067510193?check_suite_focus=true) we can see that the runner mask the secret:
![image](https://user-images.githubusercontent.com/6507291/111148818-61f08380-858c-11eb-896d-51cec1bea275.png)

Reply:
Nice! Just wanted to double check. Thanks!